<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Licenses</class>
 <widget class="QWidget" name="Licenses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лицензии</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/64x64/licence/ico/licence_64x64.png</normaloff>:/64x64/licence/ico/licence_64x64.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Организация</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Наименование:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="organization">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Лицензии</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="showParentDevice">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Отобразить привязку к устройствам</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeView" name="licenseView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="licenseButton">
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Лицензии </string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/64x64/licence/ico/licence_64x64.png</normaloff>:/64x64/licence/ico/licence_64x64.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveButton">
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Переместить</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/16x16/move/ico/move_16x16.png</normaloff>:/16x16/move/ico/move_16x16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="setFilterButton">
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Фильтр</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/16x16/add/ico/add_filter_16x16.gif</normaloff>:/16x16/add/ico/add_filter_16x16.gif</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearFilterButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Снять Фильтр</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/16x16/cancelDel/ico/del_filter_16x16.gif</normaloff>:/16x16/cancelDel/ico/del_filter_16x16.gif</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonUpdate">
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Обновить</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/48x48/update/ico/update_48x48.png</normaloff>:/48x48/update/ico/update_48x48.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonCopyAllInBufer">
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>Копировать всё
в буфер обмена</string>
            </property>
            <property name="icon">
             <iconset resource="../icons.qrc">
              <normaloff>:/16x16/copy/ico/copy.png</normaloff>:/16x16/copy/ico/copy.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</normaloff>:/16x16/cancelDel/ico/button_cancel_16x16 (1).png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddLicense">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/16x16/add/ico/add_16x16.png</normaloff>:/16x16/add/ico/add_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Добавить новую</string>
   </property>
   <property name="toolTip">
    <string>Добавить новую лицензию</string>
   </property>
  </action>
  <action name="actionDelLicense">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/16x16/cancelDel/ico/del_16x16.png</normaloff>:/16x16/cancelDel/ico/del_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить лицензию</string>
   </property>
  </action>
  <action name="actionEditLicense">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/16x16/edit/ico/edit_16x16.png</normaloff>:/16x16/edit/ico/edit_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="toolTip">
    <string>Редактировать лицензию</string>
   </property>
  </action>
  <action name="actionMoveLicense">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/16x16/move/ico/move_16x16.png</normaloff>:/16x16/move/ico/move_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Переместить</string>
   </property>
   <property name="toolTip">
    <string>Переместить лицензию</string>
   </property>
  </action>
  <action name="actionHistoryMoved">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/32x32/journal/ico/journal_32x32.png</normaloff>:/32x32/journal/ico/journal_32x32.png</iconset>
   </property>
   <property name="text">
    <string>История перемещений</string>
   </property>
   <property name="toolTip">
    <string>История перемещений лицензии</string>
   </property>
  </action>
  <action name="actionCopyLicenseInBufer">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/16x16/copy/ico/copy.png</normaloff>:/16x16/copy/ico/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать в буфер</string>
   </property>
   <property name="toolTip">
    <string>Копировать в буфер обмена</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
